ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -38,6 +45,12 @@ public Properties getKafkaProperties() {
 			props.put("key.serializer", "org.apache.kafka.common.serialization.ByteArraySerializer");
 			props.put("value.serializer", "org.apache.kafka.common.serialization.ByteArraySerializer");
 			props.put("max.block.ms", maxBlockMillis);
+			props.put("linger.ms", lingersMs);
```

Comment:
Does adding optionals to the properties work with kafka? Does it use the default if the optional is empty?